src/utils/roomVersion.ts

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2025 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * Room versions strings that we know about and do not use hydra semantics.
+ */
+const PRE_HYDRA_ROOM_VERSIONS = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11"];
+
+/**
+ * Checks if the given room version is one where new "hydra" power level
+ * semantics (ie. room version 12 or later) should be used
+ * (see https://github.com/matrix-org/matrix-spec-proposals/pull/4289).
+ * This will return `false` for versions that are known to the js-sdk and
+ * do not use hydra: any room versions unknown to the js-sdk (experimental or
+ * otherwise) will cause the function to return true.
+ *
+ * @param roomVersion - The version of the room to check.
+ * @returns `true` if hydra semantics should be used for the room version, `false` otherwise.
+ */
+export function shouldUseHydraForRoomVersion(roomVersion: string): boolean {
+    return !PRE_HYDRA_ROOM_VERSIONS.includes(roomVersion);
+}
```

Comment:
As per https://github.com/element-hq/element-meta/issues/2921 and prior discussions, we should be consistent between the clients. This approach also means a bunch of unstable (MSC) room versions will get wrongly tagged as hydra-enabled when they are not.

Reply:
Annoyingly, this would have been handled by the room capabilities stuff which failed MSC. https://github.com/matrix-org/matrix-spec-proposals/pull/3244

Reply:
Agreed, it is probably better to be consistent: have kept the function externally the same but changed the check to only consider the hydra versions.